src/cmap/commands.ts

```diff
@@ -317,6 +317,9 @@ export class OpQueryResponse {
   bsonRegExp?: boolean;
   index?: number;
 
+  /** moreToCome is an OP_MSG only concept */
+  moreToCome = false;
```

Comment:
why was this necessary?  I'd prefer to keep our existing `!('moreToCome' in response)` check than add an OpMsg-specific concept to our QpQuery classes.

also - it shouldn't be necessary on the request, right?

Reply:
It is not necessary, but neither is the `in` check, which is artificially introduced by typescript. Since OP msg vs query isn't something our driver code generally needs to discriminate it would be ideal if we could create an interface that both classes implement, then functions that do not care which protocol is being used can depend on that interface. 

That interface idea would be nice but a bit more scope than is appropriate for one condition improvement. I do not feel too strongly.